src/components/VerticalTabs/VerticalTabs.tsx

```diff
@@ -0,0 +1,209 @@
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'react-peek/prop-types';
+import { lucidClassNames } from '../../util/style-helpers';
+import {
+	findTypes,
+	getFirst,
+	omitProps,
+	Overwrite,
+	StandardProps,
+} from '../../util/component-types';
+import { buildModernHybridComponent } from '../../util/state-management';
+import * as reducers from './VerticalTabs.reducers';
+import {
+	IVerticalListMenuItemProps,
+	VerticalListMenuDumb as VerticalListMenu,
+} from '../VerticalListMenu/VerticalListMenu';
+
+const cx = lucidClassNames.bind('&-VerticalTabs');
+
+const { string, number, bool, func } = PropTypes;
+
+/** Vertical Tabs Tab Child Component */
+interface IVerticalTabsTabProps extends StandardProps {
+	/** Determines if the Tab is selected */
+	isSelected?: boolean;
+
+	/** Custom title for the tab */
+	Title?: string;
+}
+
+const Tab = (_props: IVerticalTabsTabProps): null => null;
+
+Tab.displayName = 'VerticalTabs.Tab';
+Tab.peek = {
+	description: `
+        Content that will be rendered in a tab. Be sure to nest a Title
+        inside each Tab or provide it as a prop.
+    `,
+};
+Tab.propName = 'Tab';
+Tab.propTypes = {
+	isSelected: bool`
+        Determines if the Tab is selected.
+    `,
+};
+
+/** Vertical Tabs Title Child Component */
+interface IVerticalTabsTitleProps extends StandardProps {}
+
+const Title = (_props: IVerticalTabsTitleProps): null => null;
+
+Title.displayName = 'VerticalTabs.Title';
+Title.peek = {
+	description: `
+        Titles can be provided as a child or prop to a Tab.
+    `,
+};
+Title.propName = 'Title';
+
+/** Vertical Tabs Component */
+interface IVerticalTabsPropsRaw extends StandardProps {
+	/** Custom Tab component (alias for `VerticalTabs.Tab`) */
+	Tab?: React.ReactNode;
+
+	/** Custom Title component (alias for `VerticalTabs.Title`) */
+	Title?: React.ReactNode;
+
+	/** Indicates which of the \`VerticalTabs.Tab\` children is currently
+    selected */
+	selectedIndex?: number;
```

Comment:
I think this should be marked as required since it has a default prop

Reply:
Done! Thx!

Reply:
Done! Good catch.